prow/cmd/checkconfig/main.go

```diff
@@ -125,11 +128,20 @@ func getAllWarnings() []string {
 	return all
 }
 
-func (o *options) Validate() error {
+func (o *options) DefaultAndValidate() error {
 	allWarnings := getAllWarnings()
 	if o.configPath == "" {
 		return errors.New("required flag --config-path was unset")
 	}
+
+	if o.prowYAMLPath != "" && o.prowYAMLRepoName == "" {
+		return errors.New("--prow-yaml-repo-path requires --prow-yaml-repo-name to be set")
+	}
+	if o.prowYAMLRepoName != "" {
+		if o.prowYAMLPath == "" {
+			o.prowYAMLPath = fmt.Sprintf("/home/prow/go/src/github.com/%s/.prow.yaml", o.prowYAMLRepoName)
```

Comment:
How common is this sort of path?

Reply:
This is what you get from the decoration utils, so rather common I would say

Reply:
https://github.com/kubernetes/test-infra/blob/0d180418583d65518a976b339219861ac5fafb5b/prow/pod-utilities.md